src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/services/FileService.kt

```diff
@@ -1,6 +1,8 @@
 package uk.gov.justice.digital.hmpps.riskprofiler.services
 
+import uk.gov.justice.digital.hmpps.riskprofiler.datasourcemodel.FileType
+
 interface FileService {
-  fun getLatestFile(fileLocation: String): PendingFile?
+  fun getLatestFile(fileLocation: String, fileType: FileType?): PendingFile?
   fun deleteHistoricalFiles(fileLocation: String)
```

Comment:
Possibly for another time, but can we get rid of this interface and the 'DefaultFileService' entirely now? I guess this pre-dated the convention of using localstack locally

Reply:
I think so. Just didn't want to make too many changes in one PR.